lxl-web/src/lib/components/find/FacetGroup.svelte

```diff
@@ -17,6 +19,9 @@
 	import BiCheckSquareFill from '~icons/bi/check-square-fill';
 	import BiSquare from '~icons/bi/square';
 	import BiInfo from '~icons/bi/info-circle';
+	import type { Writable } from 'svelte/store';
+
+	const matomoTracker: Writable<MatomoTracker> = getContext('matomo');
```

Comment:
I think it would be preferable to implement a `getTracker` util function (which in turn uses `getContext('matomo')` internally) – to mitigate misspellings of the matomo context string and remove the need for importing the `Writable` and `MatomoTracker` types each time the tracker is used.

Reply:
Good suggestion. Hope the latest commit does it.